wp-content/plugins/core/src/Blocks/Types/Spacer/Spacer_Model.php

```diff
@@ -0,0 +1,19 @@
+<?php
```

Comment:
It's probably still worth this block supporting the custom classes and definitely the anchor capability like the other blocks and core blocks do.

Reply:
Another thought for the future, but we will need to do some thinking around anchor links and how that plays out for focus/a11y if we plan to use a block to provide spacing between blocks. And within that convo there are some fun gotchas I experienced on Harvard when using bodylock, so we should chat about that at a future date and I can share a couple of solutions.